lib/index.js

```diff
@@ -14,5 +14,6 @@ for(const module of contextModules) {
   for(const c in contextConstants) {
     const contextUrl = contextConstants[c];
     jsonLdDocumentLoader.addStatic(contextUrl, contexts.get(contextUrl));
+    exports[c] = contextUrl;
```

Comment:
I'm not sure if this is exactly what is needed for two reasons:
 
1) some context modules generically export the context URL as `CONTEXT_URL` as here: https://github.com/digitalbazaar/cit-context/blob/master/js/constants.js#L4

`did-context` exports it's URL as `DID_CONTEXT_URL`: https://github.com/digitalbazaar/did-context/blob/main/js/constants.js#L3

2) I believe we decided that these context constants should have a version such as `CIT_CONTEXT_V1_URL` which makes it easy to reason about when `CIT_CONTEXT_V2_URL` comes out.

How do we want to reconcile these items?

We had a previous conversation where folks expressed a preference for using the generic `CONTEXT_URL` export format.

cc: @dmitrizagidulin @davidlehn @dlongley